myapp/app/controllers/application_controller.rb

```diff
@@ -22,4 +22,10 @@ def logged_in_user
 
     redirect_to '/login', flash: { notice: t('flash_msgs.no_login') }
   end
+
+  def admin_user?
+    return false if admin?
```

Comment:
It is not intuitive for a method named "admin?" to return false when the user operating the method is an admin, so it should be avoided. It is better to return true.

Reply:
Thanks for the comment.
I just made a mistake and it should return true here!
Ijust fixed it in this commit [d05ffc4](https://github.com/Fablic/training/pull/1575/commits/d05ffc4b228b1e72b4faf77de2b3a0a6c0610ead)
Please kindly check it~